webapp/.env.template

```diff
@@ -1,5 +1,6 @@
 SENTRY_DSN_WEBAPP=
 COMMIT=
 PUBLIC_REGISTRATION=false
+INVITE_REGISTRATION=true
```

Comment:
Haven’t we agreed that this is `false` in development? @ulfgebhardt @Mogge 

Otherwise I have to change it in my code …

PS: It’s not defined in the backend yet: https://github.com/Ocelot-Social-Community/Ocelot-Social/blob/3c908e147101021676cc421c8195387b39c53de8/backend/.env.template